packages/multicast_dns/pubspec.yaml

```diff
@@ -2,10 +2,11 @@ name: multicast_dns
 description: Dart package for performing mDNS queries (e.g. Bonjour, Avahi).
 repository: https://github.com/flutter/packages/tree/main/packages/multicast_dns
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+multicast_dns%22
-version: 0.3.2
+version: 0.3.3
 
 environment:
   sdk: ">=2.12.0 <3.0.0"
+  flutter: ">=2.5.0"
```

Comment:
You should not be making a package require Flutter just to use a Dart feature. The line just above allows setting a minimum Dart SDK version.

Reply:
makes sense, updated.